package.json

```diff
@@ -68,7 +687,7 @@
         "controlType": "textbox"
       },
       {
-        "key": "DEPART_DATE",
+        "key": "departDate",
```

Comment:
The format of this field should be validated in the index.js and trigger an error if the format is not valid.
Hint use RegExp